<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exit</class>
 <widget class="QWidget" name="exit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">exit:{
opacity :1;
background-color: rgba(255, 255, 255, 0);}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QFrame" name="frame_exit_main">
     <property name="minimumSize">
      <size>
       <width>358</width>
       <height>228</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>358</width>
       <height>228</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
#frame_exit_main{   
border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 174, 254, 255), stop:1 rgba(149, 209, 214, 255));
 border-top:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(84, 177, 239, 255), stop:1 rgba(255, 182, 183, 255));
    border-bottom:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(84, 177, 239, 255), stop:1 rgba(255, 182, 183, 255));
    border-right:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(84, 177, 239, 255), stop:1 rgba(255, 182, 183, 255));
	border-left:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(84, 177, 239, 255), stop:1 rgba(255, 182, 183, 255));
    border-top-right-radius:10px;
    border-bottom-right-radius:10px;
	border-top-left-radius:10px;
    border-bottom-left-radius:10px;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_exit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QFrame" name="frame_exit_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
border-bottom: 1px solid  rgb(161, 161, 161); 
border-radius:10px;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_exit">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>340</width>
                <height>4</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_exit_2_2">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{border:none;border-radius:12px;color: rgb(255, 255, 255);border-radius:8px; } QPushButton:hover{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 174, 254, 255), stop:1 rgba(149, 209, 214, 255));
}

</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../pic/icon.qrc">
                <normaloff>:/icon/icon/exit_black.png</normaloff>:/icon/icon/exit_black.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_label_exit">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>330</width>
             <height>120</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#frame_label_exit{;
	background-color: rgb(255, 255, 255);
border-radius:15px}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_exit">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_exit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">padding-top:15px</string>
              </property>
              <property name="text">
               <string>关闭提醒</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QFrame" name="frame_exit_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignHCenter">
                <widget class="QRadioButton" name="radioButton_exit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>本墨字语</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>最小化窗口    </string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QRadioButton" name="radioButton_exit_2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>本墨字语</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>退出諒笙播放</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_exit" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_exit{border:none;border-radius:10px;
border-top:1px solid   qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(84, 177, 239, 255), stop:1 rgba(255, 182, 183, 255)); }</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_exit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>本墨字语</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>不再提醒</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_exit_ok">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>本墨字语</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{;color: rgb(255, 255, 255);border-radius:8px;border: 1px solid  rgb(53, 53, 53); } QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 174, 254, 255), stop:1 rgba(149, 209, 214, 255));
}
</string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/icon.qrc"/>
 </resources>
 <connections/>
</ui>
